src/pages/Main.tsx

```diff
@@ -18,27 +18,31 @@ const mainBlock = classnames(
 )
 
 export default function () {
-  const { lg } = useBreakpoints()
-
   return (
     <div className={mainBlock}>
       <ProofsCard />
-      <CardSeparator
-        numberOfLines={3}
-        gradient="accent-to-secondary"
-        vertical={!lg}
-      />
+      <>
```

Comment:
Better to use `useBreakpoint` here, because only one prop differs

Reply:
Returned